README.md

```diff
@@ -4,8 +4,64 @@ This is a [kubernetes addon](https://github.com/mesosphere/kubeaddons) repositor
 
 # Using and Releasing `kubeaddons-kommander` Repository
 
-This repository follows the exact same rules that [`kubernetes-base-addons`](https://github.com/mesosphere/kubernetes-base-addons) does, please check documentation over there.
+This repository mostly follows the same rules that [`kubernetes-base-addons`](https://github.com/mesosphere/kubernetes-base-addons) does. Please familiarize yourself with that documentation, and then see the differences below.
 
 - [README.md](https://github.com/mesosphere/kubernetes-base-addons/blob/master/README.md)
 - [RELEASE.md](https://github.com/mesosphere/kubernetes-base-addons/blob/master/RELEASE.md)
 - [CONTRIBUTING.md](https://github.com/mesosphere/kubernetes-base-addons/blob/master/CONTRIBUTING.md)
+
+## Deviations from `kubernetes-base-addons` release process
+
+Kommander being a slightly different type of repository than `kubernetes-base-addons`, some details in releasing kommander are different.
+
+- Pre-releases are only tagged with `testing-` tags, no `stable-` tags for pre-releases. The `-rc` that will acutally be the stable GA release will be _also_ tagged with `stable-` tag (and w/o `-rc` suffix)
+- Kommander follows a more semver style versioning and is setup to be able to support multiple major and minor versions at the same time.
+
+## Step by step guides to release Kommander versions (pre-releases, latest and previously released)
+
+Only certain people are allowed to push to `testing` and `stable` branches. If you are asked to do a release, but dont have permissions to push, please ask your manager.
+
+### Tag/release pre-releases and update update SOAK
+
+1. fetch latest repo state: `git fetch`
+1. checkout `testing` branch: `git checkout testing`
+1. _merge_ `origin/master` into `testing` and push updated `testing` branch: `git merge origin/master && git push`
```

Comment:
Are you italicizing merge for a reason?
```suggestion
1. merge `origin/master` into `testing` and push updated `testing` branch: `git merge origin/master && git push`
```

Reply:
yeah, rebasing or squashing would screw up github history in these cases, so its important to use `git merge`.

Reply:
no i understand why you're using `git merge`, my question was why stress the word 'merge' in your instructions. Not sure it conveys "merge, not squash or rebase". Probably the way to make it most clear is not even explain but just list the commands to run as they seem clear enough:
```
1. git fetch
1. git checkout testing
1. git merge origin/master && git push
```